config.ini

```diff
@@ -102,37 +94,66 @@ Phase=L1
 DeviceInstance=43
 AcPosition=1
 
-[TEMPLATE]
-#CUST_SN = 12345678
-#CUST_API_PATH= cm?cmnd=STATUS+8
-#CUST_POLLING = 2000
-#CUST_Total= StatusSNS/ENERGY/Total
-#CUST_Total_Mult = 1
-#CUST_Power= StatusSNS/ENERGY/Power
-#CUST_Power_Mult = 1
-#CUST_Voltage=  StatusSNS/ENERGY/Voltage
-#CUST_Current=  StatusSNS/ENERGY/Current
+################## TEMPLATES #####################
+####Rearrange and customite Templates as necessary
+# 0=AC input 1; 1=AC output; 2=AC output 2
 
+[TEMPLATE0]
+## Tasmota Example
+## leave empty if no authentication is required
+Username = 
+Password = 
+Host=172.16.1.1
+CUST_SN = 12345678
+CUST_API_PATH= cm?cmnd=STATUS+8
+CUST_POLLING = 2000
+CUST_Total= StatusSNS/ENERGY/Total
+CUST_Total_Mult = 1
```

Comment:
Auch das ist nicht selbsterklärend.

Reply:
nein ist es auch nicht, habe den Link zur Shelly bzw Tasmota Doku hinzugefügt

Reply:
Das ist gut. Ich meine aber auch die Bedeutung der Parameter "Cust_Total" etc.. Was ist das in unserem Skript?
CUST_Total scheint ja zu sein "JSON Path to total value"
CUST_Total_Mult scheint ein Korrekturfaktor zu sein. 
Stimmt das? Da wäre eine Doku im Readme mit Beispiel(en) gut.

Reply:
ja das stimmt, steht aber hier in der Doku und die ist jetzt auch aktuell https://github.com/henne49/dbus-opendtu/blob/Mutli-Template/README.md